bin/tictactoe

```diff
@@ -1,3 +1,6 @@
 #!/usr/bin/env ruby
+require 'require_all'
+require_all 'lib'
```

Comment:
Instead of pulling in a dependency, you could do: Dir[path_to_lib].each { |f| require f }..since it's a pretty basic thing

Reply:
@ssunday I updated bin/tictactoe to include `Dir['./lib/*.rb'].each do |f|
  require f
end`

However, this doesn't require the players directory within lib. I have an additional .each loop to require the files in the player dir: `Dir['./lib/*/*.rb'].each do |f|
  require f
end`

Any thoughts on how I can combine this?

Reply:
Hmm. Can you concat the entires and then to require on them? `Dir['./lib/*/*.rb'].to_a` and concat w/ the other one and then iterate?

Reply:
@ssunday thank you! `Dir['./lib/*.rb', './lib/*/*.rb'].each do |f|
  require f
end` works!

Reply:
commit b08cd89fa1f89ff07725e912126b56faf190609d

Reply:
that's awesome!